spec/abstract-jobs.html

```diff
@@ -202,6 +212,19 @@ <h1>Execution</h1>
           in a FinalizationGroup do not necessarily hold that FinalizationGroup live.
         </p>
       </emu-note>
+      <emu-note>
+        <p>
+          Implementations are not obligated to empty WeakRefs for maximal sets
+          of non-live objects.
+        </p>
+        <p>
+          Implementations are obligated to empty WeakRefs for all objects in a
+          non-live set simultaneously. In other words, they must not empty a
```

Comment:
This is incorrect due to the way it's worded. There is a missing quantifier: as written, it's unquantified and applies to every non-live set, including one containing two unrelated objects.

Please add a quantifier, rewording it to something like "If an implementation chooses a set S in which to empty WeakRefs, it must empty WeakRefs for all objects in S simultaneously".

Reply:
That sounds like a fine clarification, I'll add it.